DevoxxClientMobile/src/main/java/com/devoxx/service/DevoxxService.java

```diff
@@ -922,6 +922,7 @@ private void loadCfpAccount(User user, Runnable successRunnable) {
     private void retrieveAuthenticatedUserSessionInformation() {
         if (isAuthenticated()) {
             retrieveNotes();
+            retrieveBadges();
```

Comment:
Should a check be added here to only retrieve badges when badges are enabled for the current conference?